docs/how_to/render_plane_manipulator.md

```diff
@@ -39,7 +39,7 @@ the rendering plane.
 	
 	![plane rendering mode](https://user-images.githubusercontent.com/1120672/207311620-0494f9cf-3059-40d4-902e-37f026556c56.png)
 
-5. Open the render plane manipulator plugin from the menu: Plugins -> napari-threedee -> render plane manipulator. You will see the render plane manipulator appear on the rendering plane.
+5. Open the render plane manipulator plugin from the menu: **Plugins -> napari-threedee -> render plane manipulator**. You will see the render plane manipulator appear in the corner of the rendering plane. You can double-click with the left mouse button (or hold down the Shift key while clicking) to reposition the manipulator. Note that depending on your versions of `napari-threedee` and napari there may be a subtle visual difference between what you see in `napari` and the screenshots/videos here.
```

Comment:
Don't you need to click the "activate" button first, before you can see the render plane manipulator? That seems like an important point to mention, in all of these tutorials

Reply:
Good point!